sound/fdk-aac/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2017 Daniel Engberg <daniel.engberg.lists@pyret.net>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fdk-aac
+PKG_VERSION=0.1.4-git-bf64cd3
```

Comment:
This is a minor nit. I know there are differing opinions here on package names containing version info. Personally, I prefer to use the date of the repo commit being referenced so I can tell which version is newer from the package/file name. Using the SHA (long or short) just doesn't convey which may be newer.

In this case: `0.1.4-20170104` would suffice.